java/src/main/java/com/google/oak/functions/weather_lookup_client/Main.java

```diff
@@ -47,9 +50,22 @@ public static void main(String[] args) throws Exception {
     // Create gRPC client stub.
     StreamingSessionGrpc.StreamingSessionStub client = StreamingSessionGrpc.newStub(channel);
 
+    // Create Oak Client.
+    GrpcStreamingTransport transport = new GrpcStreamingTransport(client::stream);
+    Result<OakClient<GrpcStreamingTransport>, Exception> oakClientCreateResult =
+        OakClient.create(transport, new InsecureAttestationVerifier());
+    if (oakClientCreateResult.isError()) {
+      throw oakClientCreateResult.error().get();
+    }
+    OakClient<GrpcStreamingTransport> oakClient = oakClientCreateResult.success().get();
+
     // Test request coordinates are defined in `oak_functions/lookup_data_generator/src/data.rs`.
     byte[] requestBody = "{\"lat\":0,\"lng\":0}".getBytes(UTF_8);
-    byte[] response = OakGrpcClient.invoke(client::stream, requestBody);
+    Result<byte[], Exception> oakClientInvokeResult = oakClient.invoke(requestBody);
```

Comment:
This code block is not very ergonomic. Perhaps it will be a bit easier if something similar to Rust's `unwrap` is implemented on the `Result` type. These 5 lines could then become `byte[] response = oakClientInvokeResult.unwrap();`